docs/csharp/language-reference/compiler-options/langversion-compiler-option.md

```diff
@@ -21,7 +21,7 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
 
 ## Arguments
 
- `option`  
+ `option`
```

Comment:
if you remove the trailing spaces here, you need to add something. Perhaps a new line?
```suggestion
 `option`
 
```

![image](https://user-images.githubusercontent.com/12971179/73223502-217d3e80-411b-11ea-8378-66d88fb172ed.png)

Reply:
And the `\` can't be used either right? It causes a loc problem?

Reply:
`\` can be used, it's not a problem for loc anymore. In this case I just went with a new line though.